alerter/NetworkAlerterStub.cs

```diff
@@ -0,0 +1,16 @@
+using System;
+
+namespace AlerterSpace
+{
+  class NetworkAlerterStub : INetworkAlerter
```

Comment:
Unused code. What is the intention of defining a class and not using it anywhere?